app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -535,7 +535,7 @@ sealed class Event {
 
     data class AutoPlaySettingChanged(val setting: AutoplaySetting) : Event() {
         enum class AutoplaySetting {
-            BLOCK_CELLULAR, BLOCK_AUDIO, BLOCK_ALL
+            BLOCK_CELLULAR, BLOCK_AUDIO, BLOCK_ALL, ALLOW_ALL
```

Comment:
@gabrielluong  I added this new type of event change as we are adding a new setting for autoplay. Is there any additional step that I have to take?